students/carlos_novoa/lesson03/mailroom.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+import sys
+
+"""
+Lesson3 - Mailroom, Part 1
+"""
+
+_donors = [
+    [
+        'Jim Halpert',
+        [
+            1.00
+        ]
+    ],
+    [
+        'Pam Beesly',
+        [
+            1000.00, 2000.00, 3000.00
+        ]
+    ],
+    [
+        'Dwight Schrute',
+        [
+            2.00, 3.00
+        ]
+    ],
+    [
+        'Michael Scott',
+        [
+            10.0, 20.00, 30.00
+        ]
+    ],
+    [
+        'Andy Bernard',
+        [
+            500.00
+        ]
+    ]
+]
+
+
+def check_for_quit(action):
+    if action.lower() == 'quit':
+        donor_app()
+        return
+    else:
+        return
+
+
+def format_donation(donation):
+    donation = float(donation)
+    donation = "{0:.2f}".format(donation)
+    return donation
+
+
+def get_donation():
+    donation = input('Donation amount: ')
+    check_for_quit(donation)
+    return donation
```

Comment:
Remember to convert `donation` to a float before storing it in your `_donors` data structure. Right now I'm seeing the following error when trying to generate a report after a new donation is added:
```
  File "./test_student_assignments.py", line 81, in <lambda>
    sorted_donors.sort(key=lambda x: sum(x[1]))
TypeError: unsupported operand type(s) for +: 'float' and 'str'
```